CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Block cursor is no longer inverted at the start/end of a selection
+- Right clicking no longer clears a selection
```

Comment:
Neither does middle mouse. We should probably phrase this to say that `LMB` is now the only button which clears the selection:

```suggestion
- Only left click without mouse mode clears the selection
```

Reply:
TBH, I like how you put it in your comment more than in your suggested change.  Slightly reworded:

```suggestion
- Clicking with the left mouse button is now the only way to clear selections
```

Reply:
That does not mention mouse mode though. So it is less accurate than my suggestion with a longer changelog entry.

It is also not true that clicking with left mouse is the only way to clear the selection.

I'm open to other suggestions though, I just tried to keep it short. My comment does not accurately state that this is about removing right/middle/other mouse button clicks but instead focuses more on mouse mode.

Reply:
Fair. Your suggestion is fine, then. Wording is a little awkward (IMHO),  but nothing's perfect, and I'd hate to block or otherwise delay this on changelog bikeshedding :P

Reply:
I mean if you have a better suggestions, I'd love to hear it. As long as it hasn't been changed yet there's still time.